docs/references/git_configuration.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_label: "Git Configuration"
+title: ""
+---
+
+# Git Configuration
+
+Since version **1.10.0**, Epinio supports Git configuration.
+
+You can provide for each git host particular settings, to clone private repositories, skip the SSL verification, or provide your own certificate bundle.
+
+A Git configuration is a Kubernetes secret with the `epinio.io/api-git-credentials: "true"` label.
+
+The available fields are:
+
+- **url (required):** the host of the git instance
+- **provider:** one of `github`, `gitlab`, `git`, `github_enterprise`, `gitlab_enterprise`
+- **username:** used during the Basic Authentication 
+- **password:** used during the Basic Authentication
+- **userOrg:** used to restrict the configuration to a specific organization/project 
+- **repo:** used to restrict the configuration to a specific repository 
+- **skipSSL:** used to skip the SSL verification 
+- **certificate:** the CA bundle to load for the SSL verification with self-signed certificates
+
+
+When importing a git repository Epinio will try to look for the existance of the most specific configuration. For example trying to clone the `https://github.com/myusername/myrepo` Epinio will first look for a configuration having the `https://github.com` *URL*, a `myusername` *userOrg* and a `myrepo` *repo*.
+If not found it will look for a configuration having the `https://github.com` *URL* and a `myusername` *userOrg*. And finally if not found Epinio will look for a configuration for the `https://github.com` *URL*.
```

Comment:
```suggestion
If not found it will look for a configuration having the `https://github.com` *URL* and a `myusername` *userOrg*. And finally it will look for a configuration having just the `https://github.com` *URL*.
```